app/views/partials/_pet_card.html.erb

```diff
@@ -1,26 +1,34 @@
 <div class="col-lg-4 col-md-6 p-0">
   <%= link_to adoptable_pet_path(pet), class: 'text-decoration-none' do %>
     <div class="card card-hover m-4" style='overflow: hidden'>
-      <% image = image_tag('coming_soon.jpg', class: 'card-img-top pet-card') %>
-      <% image = image_tag pet.images.first, class: 'card-img-top pet-card' if pet.images.attached? %>
+      <div class="position-relative d-inline-block">
+        <% image = image_tag('coming_soon.jpg', class: 'card-img-top pet-card') %>
+        <% image = image_tag pet.images.first, class: 'card-img-top pet-card' if pet.images.attached? %>
+        <%= image %>
+        <div id="adoption-status" class="position-absolute top-0 end-0 m-2">
+          <%= tag.span("Adoption Pending", class: "badge bg-info") if pet.has_adoption_pending? %>
+        </div>
+      </div>
+  <% end %>
 
-      <%= image %>
       <ul class="list-group list-group-flush">
-        <li class="list-group-item d-flex justify-content-between">
-          <h5 class="card-title text-secondary mb-0">
-            <%= pet.name %>
-          </h5>
-          <div id="adoption-status">
-            <%= tag.span("Adoption Pending", class: "badge bg-info") if pet.has_adoption_pending? %>
-          </div>
+        <li class="list-group-item d-flex justify-content-between align-items-center flex-wrap">
+          <%= link_to adoptable_pet_path(pet), class: 'text-decoration-none' do %>
+            <h3 class="card-title text-secondary mb-0">
+              <%= pet.name %>
+            </h3>
+          <% end %>
+          <% if current_user&.adopter_foster_account %>
```

Comment:
I am still using this because I find it easier to read...thoughts?

Reply:
Yes, I agree there is pros and cons to both approaches. Perhaps something to discuss broader so we can set a consistent approach through the app.